src/obr/signac_wrapper/operations.py

```diff
@@ -21,11 +22,36 @@
 
 
 class OpenFOAMProject(flow.FlowProject):
+    filtered_jobs = None
+    queried_jobs = None
+
     def print_operations(self):
         ops = sorted(self.operations.keys())
         logging.info("Available operations are:\n\t" + "\n\t".join(ops))
         return
 
+    def get_jobs(self, filter=Union[None, dict[str, str]], queries=None):
+        self.filtered_jobs = self.filter_jobs(filters=filter)
+        resulting_jobs = self.query_jobs()
+        return resulting_jobs
+
+    def filter_jobs(self, filters: dict[str, str], output=True):
+        all_jobs: list[Job] = self.queried_jobs or self.find_jobs()
+        # chain filtering of jobs
+        for key, value in filters.items():
+            all_jobs = filter(
+                lambda job: value in self.get_job_status(job)[key], all_jobs
+            )
+        self.filtered_jobs = list(all_jobs)
+        if output:
+            for job in self.filtered_jobs:
+                print(f"Found Job with {job.id=}")
+
+    def query_jobs(self, query) -> list[Job]:
```

Comment:
I am unsure if that is needed, since query should about getting information out of the jobs rather then selecting jobs.  So it would make more sense to allow queries on a list of filtered (selected?) jobs.

Reply:
I agree, the current implementation of `query_jobs` simply forwards the query to `query_impl` of `queries.py`.
(This only works because of the `__iter__` function of `OpenFOAMProject`.)